features/GGG_CFE_Integration_tests_EmploymentTests/GGG_EMP_LangleyYorke.feature

```diff
@@ -0,0 +1,47 @@
+Feature:
+  " 1. Most recent (within £60 tolerance)
+    2. Variations - MOST RECENT
+    3. 1 weekly "
```

Comment:
you can get rid of this, looks like duplicate

Reply:
i copied the test and didnt update the description